README.md

```diff
@@ -63,6 +63,16 @@ for the reference implementation
 * [Miscellaneous Docs](https://github.com/theupdateframework/python-tuf/tree/develop/docs)
 * [Python-TUF development blog](https://theupdateframework.github.io/python-tuf/)
 
+Contribution and Community
+--------
+We welcome Contributions, updates, [discrepancy reports](https://github.com/theupdateframework/python-tuf/issues) and [pull requests](https://github.com/theupdateframework/python-tuf/pulls) are welcome. This project thrives on community collaboration. Contributors are expected to adhere to our [Code of Conduct](https://github.com/theupdateframework/python-tuf/blob/develop/docs/CODE-OF-CONDUCT.md).
+
+Jump into our [Slack](https://communityinviter.com/apps/cloud-native/cncf)! Our projects are community-built and welcome collaboration. 
```

Comment:
Hi @Ayush9026 Hope you are doing well. IMO we don't need any contact details here (like Slack link). Just below that we have `Contact` section, which is the perfect place to define that and it is [already ](https://github.com/theupdateframework/python-tuf?tab=readme-ov-file#contact)there. WYT? 


Also as i said [here](https://github.com/theupdateframework/theupdateframework.io/pull/58#discussion_r1614667316), Please wait for official maintainer's review what they think, before making any changes. I hope you understand. Thank you😊👍

Reply:
Hi @h4l0gen,

Any PR doesn't get merged without an official maintainer's review, and I always mention the issue in the PR. I raised this PR with the permission of @lukpueh sir. I just wanted to thank you for your feedback and didn’t make any changes based on it yet. 
I understand the importance of patience in open source contributions. I have been contributing to different projects for some time, and I assure you that I follow the protocol of seeking maintainers' approval before raising any PR. Thank you for your understanding and support.